docs/tutorial/arguments/optional.md

```diff
@@ -39,8 +39,11 @@ In the [First Steps](../first-steps.md#add-a-cli-argument){.internal-link target
 
 Now let's see an alternative way to create the same *CLI argument*:
 
+{* docs_src/arguments/optional/tutorial000.py hl[4] *}
 
-{* docs_src/arguments/optional/tutorial001_an.py hl[5] *}
+Or, using an explicit `Typer()` instance creation:
+
+{* docs_src/arguments/optional/tutorial001_an.py hl[8] *}
```

Comment:
Here, I've created a new tutorial file (`docs_src/arguments/optional/tutorial000.py`) to better show the difference with the more simplistic `docs_src/first_steps/tutorial002.py`. The next version then (`docs_src/arguments/optional/tutorial001_an.py`) introduces both `Annotated` as well as `Typer()`.